drivers/sensor/stm32_temp/stm32_temp.c

```diff
@@ -60,15 +59,21 @@ static int stm32_temp_sample_fetch(const struct device *dev, enum sensor_channel
 
 	k_mutex_lock(&data->mutex, K_FOREVER);
 
+	rc = adc_channel_setup(data->adc, &data->adc_cfg);
+	if (rc) {
+		LOG_DBG("Setup AIN %u got %d", data->adc_cfg.channel_id, rc);
```

Comment:
If I edit this like so I will be able to save a few bytes if the user also compiled the vbat driver:

```suggestion
		LOG_DBG("Setup AIN%u got %d", data->adc_cfg.channel_id, rc);
```